Mobile/ShopLens/Droid/Activities/ShoppingCartActivity.cs

```diff
@@ -97,8 +114,11 @@ protected override void OnPause()
 
         protected override void OnStop()
         {
-            // Stop Tts Speech.
-            shopLensTts.Stop();
+            if (!talkBackEnabled && !voiceIsOff)
```

Comment:
I think we always want to call tts.stop() and we always want to create a tts object even if voice commands are turned off, because this can happen.
![image](https://user-images.githubusercontent.com/43152072/49702867-68059600-fc06-11e8-8814-007d45631e42.png)